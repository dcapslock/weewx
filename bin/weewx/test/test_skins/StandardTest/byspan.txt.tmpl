Daily Top Temperatures
#for $iterDay in $last_week.days
$iterDay.dateTime.format("%A %-d %B"): $iterDay.outTemp.max at $iterDay.outTemp.maxtime
#end for

3 hour averages over last day
DateTime,outTemp
#for $span in $last_day.spans(interval=10800)
$span.dateTime.format("%d/%m %H:%M:%S"),$span.outTemp.avg
#end for

All Day Values
#for $record in $last_day.records
$record.dateTime.format("%d/%m %H:%M:%S"),$record.outTemp
#end for