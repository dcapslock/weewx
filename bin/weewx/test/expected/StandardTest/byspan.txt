Daily Top Temperatures
Friday 27 August: 82.7F at 19:00
Saturday 28 August: 82.1F at 19:00
Sunday 29 August: 81.5F at 19:00
Monday 30 August: 80.9F at 19:00
Tuesday 31 August: 80.3F at 19:00
Wednesday 1 September: 79.8F at 19:00
Thursday 2 September: 79.1F at 19:00
Friday 3 September: 63.3F at 00:10

3 hour averages over last day
DateTime,outTemp
02/09 09:00:00,48.1F
02/09 12:00:00,62.3F
02/09 15:00:00,74.9F
02/09 18:00:00,78.5F
02/09 21:00:00,70.6F
03/09 00:00:00,56.0F
03/09 03:00:00,43.2F
03/09 06:00:00,39.5F
03/09 09:00:00,45.8F

All Day Values
02/09 11:30:00,51.7F
02/09 11:40:00,52.5F
02/09 11:50:00,53.3F
02/09 12:00:00,54.1F
02/09 12:10:00,55.0F
02/09 12:20:00,55.8F
02/09 12:30:00,56.7F
02/09 12:40:00,57.6F
02/09 12:50:00,58.4F
02/09 13:00:00,59.3F
02/09 13:10:00,60.2F
02/09 13:20:00,61.0F
02/09 13:30:00,61.9F
02/09 13:40:00,62.8F
02/09 13:50:00,63.6F
02/09 14:00:00,64.5F
02/09 14:10:00,65.3F
02/09 14:20:00,66.1F
02/09 14:30:00,66.9F
02/09 14:40:00,67.7F
02/09 14:50:00,68.5F
02/09 15:00:00,69.3F
02/09 15:10:00,70.0F
02/09 15:20:00,70.7F
02/09 15:30:00,71.4F
02/09 15:40:00,72.1F
02/09 15:50:00,72.7F
02/09 16:00:00,73.4F
02/09 16:10:00,74.0F
02/09 16:20:00,74.5F
02/09 16:30:00,75.1F
02/09 16:40:00,75.6F
02/09 16:50:00,76.1F
02/09 17:00:00,76.5F
02/09 17:10:00,76.9F
02/09 17:20:00,77.3F
02/09 17:30:00,77.7F
02/09 17:40:00,78.0F
02/09 17:50:00,78.3F
02/09 18:00:00,78.5F
02/09 18:10:00,78.7F
02/09 18:20:00,78.9F
02/09 18:30:00,79.0F
02/09 18:40:00,79.1F
02/09 18:50:00,79.1F
02/09 19:00:00,79.1F
02/09 19:10:00,79.1F
02/09 19:20:00,79.1F
02/09 19:30:00,79.0F
02/09 19:40:00,78.8F
02/09 19:50:00,78.7F
02/09 20:00:00,78.4F
02/09 20:10:00,78.2F
02/09 20:20:00,77.9F
02/09 20:30:00,77.6F
02/09 20:40:00,77.2F
02/09 20:50:00,76.8F
02/09 21:00:00,   N/A
02/09 21:10:00,76.0F
02/09 21:20:00,75.5F
02/09 21:30:00,75.0F
02/09 21:40:00,74.4F
02/09 21:50:00,73.8F
02/09 22:00:00,73.2F
02/09 22:10:00,72.6F
02/09 22:20:00,71.9F
02/09 22:30:00,71.2F
02/09 22:40:00,70.5F
02/09 22:50:00,69.8F
02/09 23:00:00,69.0F
02/09 23:10:00,68.3F
02/09 23:20:00,67.5F
02/09 23:30:00,66.7F
02/09 23:40:00,65.9F
02/09 23:50:00,65.0F
03/09 00:00:00,64.2F
03/09 00:10:00,63.3F
03/09 00:20:00,62.5F
03/09 00:30:00,61.6F
03/09 00:40:00,60.7F
03/09 00:50:00,59.9F
03/09 01:00:00,59.0F
03/09 01:10:00,58.1F
03/09 01:20:00,57.2F
03/09 01:30:00,56.4F
03/09 01:40:00,55.5F
03/09 01:50:00,54.6F
03/09 02:00:00,53.8F
03/09 02:10:00,53.0F
03/09 02:20:00,52.1F
03/09 02:30:00,51.3F
03/09 02:40:00,50.5F
03/09 02:50:00,49.7F
03/09 03:00:00,48.9F
03/09 03:10:00,48.2F
03/09 03:20:00,47.5F
03/09 03:30:00,46.8F
03/09 03:40:00,46.1F
03/09 03:50:00,45.4F
03/09 04:00:00,44.8F
03/09 04:10:00,44.2F
03/09 04:20:00,43.6F
03/09 04:30:00,43.0F
03/09 04:40:00,42.5F
03/09 04:50:00,42.0F
03/09 05:00:00,41.6F
03/09 05:10:00,41.2F
03/09 05:20:00,40.8F
03/09 05:30:00,40.4F
03/09 05:40:00,40.1F
03/09 05:50:00,39.8F
03/09 06:00:00,39.6F
03/09 06:10:00,39.3F
03/09 06:20:00,39.2F
03/09 06:30:00,39.0F
03/09 06:40:00,38.9F
03/09 06:50:00,38.9F
03/09 07:00:00,38.8F
03/09 07:10:00,38.9F
03/09 07:20:00,38.9F
03/09 07:30:00,39.0F
03/09 07:40:00,39.1F
03/09 07:50:00,39.3F
03/09 08:00:00,39.5F
03/09 08:10:00,39.7F
03/09 08:20:00,40.0F
03/09 08:30:00,40.3F
03/09 08:40:00,40.7F
03/09 08:50:00,   N/A
03/09 09:00:00,41.5F
03/09 09:10:00,41.9F
03/09 09:20:00,42.4F
03/09 09:30:00,42.9F
03/09 09:40:00,43.5F
03/09 09:50:00,44.0F
03/09 10:00:00,44.6F
03/09 10:10:00,45.3F
03/09 10:20:00,45.9F
03/09 10:30:00,46.6F
03/09 10:40:00,47.3F
03/09 10:50:00,48.0F
03/09 11:00:00,48.7F
03/09 11:10:00,49.5F
03/09 11:20:00,50.3F

